### PR TITLE
Set up basic Admin dashboard

### PR DESCRIPTION
**Pull request by Willdotwhite**
This PR also adds a config singleton to the API and hides the login button when logged in

image
image

### CODE REVIEW

**Comment by dallonf** [comment]
Made enough updates here that I think I'd like someone else to review it

**Comment by Willdotwhite** [comment]
@dallonf your changes look good to me, thank you - very clear, make a lot of sense.

I can't approve the PR because I have a commit on it, but we can either have a third party review your changes (@runny-yolk is the prime candidate) or get a verbal approval and I can merge regardless?

**Review by dallonf — APPROVED** [review]
I'm good with a verbal approval and, er, my own I guess, haha

**Pull request merged**
